data_test/tests_lpnamer/tests_integer/test_lpnamer_01/output/economy/reference_lp/master.mps

```diff
@@ -13,17 +13,17 @@ COLUMNS
     semibase R0000001 1
     transmission_line OBJROW 10000
     transmission_line R0000002 1
```

Comment:
Is is wanted to have output files ?
I know they're not added but it may be that they should not exists in the first place

Reply:
Yes they are used in `lpnamer` end to end tests (see `tests/end_to_end/lpnamer/test_lpnamerEndToEnd.py`). These are the expected output of lpnamer from the mps and txt files  directly stored in the `output` directory of the test study.
However the test produces files in the `lp` directory, we could change that with a temporary location. This point is however not related to this PR.

Reply:
Ok